src/python/turicreate/util/lambda_closure_capture.py

```diff
@@ -256,7 +258,13 @@ def visit_FunctionDef(self, node):
             # it actually shows up in the ast as a Expr.str
             # so we need to catch that and skip it
             try:
-                if type(next_node) is ast.Expr and type(next_node.value) is ast.Str:
+                # Docstrings are constants in Python 3.8 and strings in all other
+                # supported Python versions.
+                if python_version.major == 3 and python_version.minor == 8:
```

Comment:
Should this be `python_version.major == 3 and python_version.minor >= 8` to account for future Python3 versions that will also need this behavior? Presumably Python 4+ will as well but I wouldn't worry about those yet.

Reply:
That's a good idea. Done.